modules/AOP_Case_Updates/CaseUpdatesHook.php

```diff
@@ -602,7 +602,7 @@ public function sendCaseUpdate(AOP_Case_Updates $caseUpdate)
     {
         global $current_user, $sugar_config;
         $email_template = new EmailTemplate();
-        if ($_REQUEST['module'] === 'Import') {
+        if ((isset($_REQUEST['module']) ? $_REQUEST['module'] : null) === 'Import') {
```

Comment:
I think this is weird a little bit, since null will be never equal to a string.
What do you think about this?
isset($_REQUEST['module'] && $_REQUEST['module'] === 'Import'